web/src/context/CBD.svelte

```diff
@@ -76,9 +77,16 @@
   </p>
 </ContextLayerButton>
 
-<ContextLayerButton bind:show={showExistingInfra} label="Cycle infrastructure">
+<RouteNetwork />
+
+<div class="layer-group">Other</div>
```

Comment:
Slightly awkwardly, nest one of the layers in here, to achieve the desired grouping/ordering. One svelte component for both the map and controls remains useful, but means this kind of thing has to happen

Reply:
Would this awkwardness go away if we pulled the "traffic" layer and context button into a separate component? Then we could put the heading in ContextualLayers.svelte and put this new component after it.

Maybe there's a reason that's non-trivial. In any case, leaving it as is seems like no big deal.